sources/packages/web/src/components/common/OfferingForm.vue

```diff
@@ -1,6 +1,7 @@
 <template>
+  <!-- todo: ann form definition -->
   <formio-container
-    formName="educationprogramoffering"
+    formName="educationProgramOffering"
```

Comment:
Why camel case for form-name?

Reply:
since we are following camelcase in our code for variables. and if we use camel case we can avoid this too 
![image](https://user-images.githubusercontent.com/77353155/188738979-37cf590f-7b47-4480-9898-dfa43beff4a6.png). not sure if you noticed in my previous PR. actually we can use camel case for form name even though in form definition the form name is lowecase

Reply:
A couple is already following the pascal case in the previous PR in the last sprints. Since it works in the same way it is better to read 😉